CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ if (TESTS)
 endif ()
 
 add_subdirectory(Tools)
+add_subdirectory(Python)
```

Comment:
Please make this conditional on a flag (default to true, but let people turn it off).